docs-en/chapter_preface/suggestions.md

```diff
@@ -2,223 +2,225 @@
 
-The preliminaries for running the code are divided into three main steps.
+Setting up to run the code involves three main steps.
 
-**Step 1: Install the local programming environment**. Please refer to [Appendix Tutorial](https://www.hello-algo.com/chapter_appendix/installation/) for installation, or skip this step if already installed.
+**Step 1: Install a local programming environment**. Follow the [tutorial](https://www.hello-algo.com/chapter_appendix/installation/) in the appendix for installation, or skip this step if already installed.
 
-**Step 2: Clone or download the code repository**. If [Git](https://git-scm.com/downloads) is already installed, you can clone this repository with the following command.
+**Step 2: clone or download the code repository**. Visit the [GitHub Repository](https://github.com/krahets/hello-algo).
+
+If [Git](https://git-scm.com/downloads) is installed, use the following command to clone the repository:
 
 ```shell
 git clone https://github.com/krahets/hello-algo.git
 ```
 
-Of course, you can also in the location shown in the figure below, click "Download ZIP" directly download the code zip, and then in the local solution.
+Alternatively, you can also click the "Download ZIP" button at the location shown in the figure below to directly download the code as a compressed ZIP file. Then, you can simply extract it locally.
 
-![Clone repository with download code](suggestions.assets/download_code.png)
+![Cloning Repository and Downloading Code](suggestions.assets/download_code.png)
 
-**Step 3: Run the source code**. As shown in the figure below, for the code block labeled with the file name at the top, we can find the corresponding source code file in the `codes` folder of the repository. The source code files can be run with a single click, which will help you save unnecessary debugging time and allow you to focus on what you are learning.
+**Step 3: Run the source code**. As shown in the figure below, for the code block labeled with the file name at the top, we can find the corresponding source code file in the `codes` folder of the repository. These files can be executed with a single click, which will help you save unnecessary debugging time and allow you to focus on learning.
 
-![Code block with corresponding source file](suggestions.assets/code_md_to_repo.png)
+![Code Block and Corresponding Source Code File](suggestions.assets/code_md_to_repo.png)
 
-## Growing Together In Questioning And Discussion
+## Learning Together in Discussion
 
-While reading this book, please don't skip over the points that you didn't learn. **Feel free to ask your questions in the comment section**. We will be happy to answer them and can usually respond within two days.
+While reading this book, please don't skip over the points that you didn't learn. **Feel free to post your questions in the comment section**. We will be happy to answer them and can usually respond within two days.
```

Comment:
"While reading this book, please don't skip over the points that you didn't learn. **Feel free to post your questions in the comment section**. We will be happy to answer them and can usually respond within two days."
can be changed to ->
"While progressing through this book, please don't overlook sections that may be challenging. **Feel free to post your queries in the comment section**. We're eager to assist and typically respond within two days."